components/org.wso2.carbon.identity.api.user.approval/org.wso2.carbon.identity.api.user.approval.common/src/main/java/org/wso2/carbon/identity/api/user/approval/common/UserApprovalServiceHolder.java

```diff
@@ -17,18 +17,26 @@
 package org.wso2.carbon.identity.api.user.approval.common;
 
 import org.wso2.carbon.humantask.core.TaskOperationService;
+import org.wso2.carbon.identity.workflow.engine.ApprovalEventService;
 
 /**
  * Service holder class for user approvals.
  */
 public class UserApprovalServiceHolder {
 
     private static TaskOperationService taskOperationService;
+    private static ApprovalEventService approvalEventService;
 
     public static void setTaskOperationService(TaskOperationService taskOperationService) {
+
         UserApprovalServiceHolder.taskOperationService = taskOperationService;
     }
 
+    public static void setApprovalEventService(ApprovalEventService approvalEventService1) {
```

Comment:
add method comments to  public methods. Fix other places as well.

Reply:
Addressed with this commit e7dc2ae0d29fe0137b37397251c8596a67ec212a